pyvlx/api/frames/frame_activate_scene.py

```diff
@@ -1,7 +1,7 @@
 """Module for sending command to gw."""
 from enum import Enum
 
-from pyvlx.const import Command, Originator, Priority, Velocity
+from ...const import Command, Originator, Priority, Velocity
```

Comment:
Just wondering, is this `...` syntax recommended? Or might this have some side effects?

Reply:
ill fix this - just to avoid side effects
applies to all imports from parent(s). i will make those absolute. but use relative for sub and same directories

from pyvlx.log import PYVLXLOG
from .api_event import ApiEvent
from .frames import ..